### PR TITLE
Convert docs from the Unicode Tools classic Google Site to Markdown files

### PR DESCRIPTION
**Pull request by markusicu**
From https://sites.google.com/site/unicodetools/home with (hopefully) minimal contents changes.

Before this PR is merged, see the converted docs at https://github.com/markusicu/unicodetools/tree/site_to_docs_md/docs

### CODE REVIEW

File: docs/changing-ucd-properties.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+The properties built directly from files have constants defined in
+UCD_Types/Names. You normally have to add a constant and update a LIMIT value in
+UCD_Types, and then add a long name and a short name in UCD_Names.
+
+**Some of this is outdated**; see also the [New Unicode Properties](newunicodeproperties.md) page.
+Sometimes you have to make changes in both the old & new mechanisms.
+
+Examples:
+
+// line break
```
Comment (Manishearth) [inline docs/changing-ucd-properties.md:16]: These should be encased in code blocks (triple backticks)

**Review by Manishearth — CHANGES_REQUESTED** [review]
There's a lot of stuff that should be encased in code blocks. Happy to make those changes myself.

**Comment by srl295** [comment]
> There's a lot of stuff that should be encased in code blocks. Happy to make those changes myself.

you could PR the PR…

**Comment by markusicu** [comment]
> There's a lot of stuff that should be encased in code blocks.

I know. And `inline code style`, and some other stuff. That's why the PR is still "draft" :-)

I have gotten only to two files so far for fixing text styles. I plan to keep amending my third commit. Stay tuned...

File: docs/changing-ucd-properties.md (hunk 1 above)
Comment (markusicu) [reply docs/changing-ucd-properties.md]: This file is done now. On to the next one.

**Comment by markusicu** [comment]
FYI When I redo code blocks, I copy the original text from the Google Site, rather than trying to clean up what the converter wrote.

**Comment by markusicu** [comment]
I think I made it through the whole site / set of docs. Please review.

My goal is not perfection. My goal is to preserve the contents of the Google Site, and as much of the text styles as makes sense (and under Markdown constraints), so that we can abandon the Site.

(Going forward, every time we run a tool we should peek at its docs page and see if still matches reality.)

**Comment by markusicu** [comment]
Hi guys, could I please get at least a rubber-stamp here?
@Manishearth I went through all of the pages. Do you see more glaring issues? Or can we merge this and then treat further issues in follow-up changes?

**Review by srl295 — APPROVED** [review]
RSLGTM (had to get more ink for it!)

I assume the image links will work once all is merged.

Thanks for your work on this.

**Comment by markusicu** [comment]
> I assume the image links will work once all is merged.

Which links don't work? I can see the 4 images on https://github.com/markusicu/unicodetools/blob/site_to_docs_md/docs/unicodejsps/index.md

(Let me know if there is Markdown syntax for making images display smaller.)

**Comment by srl295** [comment]
@markusicu they just hadn't worked in the rich preview (though the syntax seemed correct). probably a bug

**Comment by macchiati** [comment]
Thanks Markus!

**Review by Manishearth — APPROVED** [review]
Looks great to me!

**Pull request merged**